dash/backend/src/modules/falco/dao/falco.dao.ts

```diff
@@ -17,6 +18,95 @@ export class FalcoDao {
         private readonly databaseService: DatabaseService,
     ) {}
 
+    private async buildFalcoLogQuery(
```

Comment:
I'd prefer we either figure out why calling the function is failing or remove the dead code/comments.

Reply:
Done in https://github.com/m9sweeper/m9sweeper/pull/164